cves/CVE-2011-2729.yml

```diff
@@ -116,10 +116,10 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
-  contest:
+  answer: This was identified by Wilfried Weissmann on 20 July 2011 and made public on 12 August 2011. Affects 5.5.32-5.5.33  No specifics on how it was found were listed.
```

Comment:
Who is Wilfried Weissman? Is he a part of the team or an external security researcher?

Reply:
The page didn't specify who Wilfried Weissman I will do more research and post an update when I find out.